aws/aws.go

```diff
@@ -0,0 +1,42 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+var regions []string
+
+// GetAllResources - Lists all aws resources
+func GetAllResources() []string {
+	regions = []string{
+		"us-east-1", "us-east-2", "us-west-1", "us-west-2", "ca-central-1",
+		"eu-west-1", "eu-central-1", "eu-west-2", "ap-southeast-1", "ap-southeast-2",
+		"ap-northeast-2", "ap-northeast-1", "ap-south-1", "sa-east-1",
+	}
+
+	var resources []string
+	for _, region := range regions {
+		session, _ := session.NewSession(&awsgo.Config{
```

Comment:
I'm guessing the second parameter is an error object. If so, do not ignore it. Although it's verbose, the idiomatic code style in Go is for every function (including this `GetAllResources` function) to return a value (e.g., `[]string` and an error):

```go
func GetAllResources() ([]string, error) {
  // ...
  session, err := session.NewSession(&awsgo.Config{ ... })
  if err != nil {
    return nil, errors.WithStackTrace(err)
  }
}
```